packager/app/stream_descriptor.cc

```diff
@@ -206,6 +210,14 @@ base::Optional<StreamDescriptor> ParseStreamDescriptor(
             base::SplitString(iter->second, ";", base::TRIM_WHITESPACE,
                               base::SPLIT_WANT_NONEMPTY);
         break;
+      case kDashOnlyField:
+        if (iter->second == "true")
```

Comment:
Are you ok with using "1" or "0" to be consistent with "skip_encryption" in line 170?

We may also support all of them.

Reply:
I am fine with "1" or "0". Will make the change.